swarm/CHANGELOG.md

```diff
@@ -12,12 +12,12 @@
 - Rename `NetworkBehaviourAction` to `ToSwarm`.
   A deprecated type-alias is provided to ease the transition.
   The new name is meant to better indicate the message-passing relationship between `Swarm` and `NetworkBehaviour`.
-  See [PR XXXX].
+  See [PR 3658].
```

Comment:
Whoops, sorry about that!

Reply:
Not a problem. Could have easily occurred to me too.